src/eventsource.js

```diff
@@ -201,7 +210,18 @@ $.EventSource.prototype = {
             return handler( this, eventArgs || {} );
         }
         return undefined;
+    },
+
+    /**
+     * Set an event name as being disabled, and provide an optional error message
+     * to be printed to the console
+     * @param {String} eventName - Name of the event
+     * @param {String} [errorMessage] - Optional string to print to the console
+     */
+    rejectEventHandler(eventName, errorMessage = ''){
```

Comment:
I don't think this needs to be added to the public API; let's mark it as "private".